code/hud/hudtarget.cpp

```diff
@@ -5533,6 +5533,16 @@ void HudGaugeWeaponEnergy::initBitmaps(char *fname)
 	Energy_bar.first_frame = bm_load_animation(fname, &Energy_bar.num_frames);
 	if ( Energy_bar.first_frame < 0 ) {
 		Warning(LOCATION,"Cannot load hud ani: %s\n", fname);
+	} else {
```

Comment:
thanks! Although in this case just diff'ing vs the previous commit should be fine?
```
git diff -w HEAD^
```